packages/backend/src/clients/Google/GoogleDriveClient.ts

```diff
@@ -266,12 +266,6 @@ export class GoogleDriveClient {
             throw new Error('Google Drive is not enabled');
         }
 
-        const auth = await this.getCredentials(refreshToken);
```

Comment:
Moved this gsheet api specific methods to another method that is going to be reused for pivot and not pivot tables